code/api/src/modules/user/model.js

```diff
@@ -14,12 +14,22 @@ module.exports = function(sequelize, DataTypes) {
     },
     role: {
       type: DataTypes.TEXT
+    },
```

Comment:
These three properties are the newly added features that we want our user to be able to edit, thank you for putting these into the already existing `User` data. This will make retrieving the data to display on the front end painless, thank you!